roles/xdebug/tasks/main.yml

```diff
@@ -19,13 +19,13 @@
       state: link
     notify: reload php-fpm
 
-  when: xdebug_remote_enable | bool
+  when: xdebug_mode is not match("off")
```

Comment:
🤔  their docs for `off` says:

> Nothing is enabled. Xdebug does no work besides checking whether functionality is enabled. Use this setting if you want close to 0 overhead.

Maybe we can just *always* template the config file? If it exists but the mode is `off`, then nothing is enabled anyway. Seems much easier than removing/adding the file.

Reply:
That is probably a good idea. In 04d4bc3 I also skipped removing the conf file if `mode == "off"`. Not sure what impact that will have on prod environments?

Reply:
The xdebug role is only included in `dev.yml` and not the production `server.yml` so we should be fine here 😄